### PR TITLE
Update airbrake version to be compatible with latest rails and ruby

### PR DESCRIPTION
**Pull request by cintamani**
From: https://eaflood.atlassian.net/browse/RUBY-1060

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -16,7 +16,7 @@ git:
 
 before_install:
   - export TZ=UTC
-  - gem install -v 1.17.3 bundler --no-rdoc --no-ri
+  - gem install bundler --no-rdoc --no-ri
```
Comment (cintamani) [inline .travis.yml]:
```suggestion
  - gem install bundler --no-document
```

**Comment by cintamani** [comment]
Seems like the newest version of Airbrake does not have the close need from the failures on travis. Leaving it here to gather the rest of the team opinion on this.

**Comment by Cruikshanks** [comment]
I think if we can recreate throwing an error during a rake task and confirming it gets logged in Errbit, then we should be good to stop using that aspect of the gem. This was the original reason for including it (and creating this gem); we were missing notifications in Errbit from failed rake tasks because Airbrake wasn't being given a chance to close.

**Comment by Cruikshanks** [comment]
I opened another PR based on this one to look at the issue. Go here for details https://github.com/DEFRA/defra-ruby-alert/pull/9#issuecomment-638487028. But after some playing around I'm happy that we should be removing our `at_exit` hook logic and that this will fix our issue with this gem.

Unfortunately, I ran out of time this evening to actually make the change to my PR. But will do it first thing tomorrow!

**Review by Cruikshanks — APPROVED** [review]

**Pull request merged**